Dockerfile

```diff
@@ -15,12 +15,10 @@ ENV TF_IN_AUTOMATION="yes"
 ENV CGO_ENABLED=0
 
 # Install dependencies
-RUN apk add --update docker bash git openrc openssh openssl python3
-
+#
 # We run Docker as Docker-out-of-Docker (DooD). DooD is a solution where you run the Docker CLI inside a container,
-# and connect it to the host’s Docker by virtue of mount the /var/run/docker.sock into the container. Docker should be
-# started on container startup.
-RUN rc-update add docker boot
+# and connect it to the host’s Docker by virtue of mount the /var/run/docker.sock into the container.
+RUN apk add --update docker bash git openrc openssh openssl python3
```

Comment:
why do we need to add docker at all?

Reply:
we basically only need the cli

Reply:
to do what?

Reply:
We are using "docker out of docker" in order to run the forensics https://github.com/mineiros-io/build-tools/blob/master/Makefile#L84

Reply:
yes, but this is not in the buil-tools container... this a a snyk container running a test on the container...  afaiu?

Reply:
yep but it's run inside the build tools container

Reply:
Where do we run it there?

Reply:
We do not only use it for snyk but also for some tests, e.G. https://github.com/mineiros-io/terraform-aws-ecr/blob/master/test/ecr_repository_test.go#L99

This is also where in it crucial afaik. Actually, Snyk runs DooD inside its own container that isn't run by build-tools.

Reply:
As you can see, for the tests it is mandatory.

Reply:
Imho, a good solution would be to install the CLI only.

Reply:
Is the ecr thing running the Client or using the api directly?

Reply:
approved ... we shoudl check how to only install docker cli and if the docker go code actually needs the binary..?

Reply:
Let me look into that before we merge this.

Reply:
too late ;) ... can be improved in next steps... ;)